Wave1_data/4.processing_profiled_features/scripts/0.merge_sc.py

```diff
@@ -24,28 +25,70 @@
 # In[2]:
 
 
+# check if in a jupyter notebook
+try:
+    cfg = get_ipython().config
+    in_notebook = True
+except NameError:
+    in_notebook = False
+
+if not in_notebook:
+    print("Running as script")
+    # set up arg parser
+    parser = argparse.ArgumentParser(description="Segment the nuclei of a tiff image")
+
+    parser.add_argument(
+        "--input_dir",
+        type=str,
+        help="Path to the input directory containing the tiff images",
+    )
+
+    args = parser.parse_args()
+    input_dir = pathlib.Path(args.input_dir).resolve(strict=True)
+else:
+    print("Running in a notebook")
+    input_dir = pathlib.Path(
+        "../../3.cellprofiling/analysis_output/W0052_F0001"
+    ).resolve(strict=True)
+
+
+# In[3]:
+
+
 # type of file output from CytoTable (currently only parquet)
 dest_datatype = "parquet"
 
-# s1lite directory
-source_dir = pathlib.Path("../../3.cellprofiling/analysis_output")
 # directory where parquet files are saved to
 output_dir = pathlib.Path("../data/converted_data")
 output_dir.mkdir(exist_ok=True, parents=True)
 
 
+# In[4]:
+
+
+# get the .sqlite file from the input directory
+sqlite_file = list(input_dir.glob("*.sqlite"))[0]
+dest_path = output_dir / str(sqlite_file.parent).split("/")[-1]
+dest_path.mkdir(exist_ok=True, parents=True)
+dest_path = dest_path / f"{sqlite_file.stem}.{dest_datatype}"
+print(f"Destination path: {dest_path}")
+
+
 # ## set config joins for each preset
 
-# In[3]:
+# In[5]:
 
 
 # preset configurations based on typical CellProfiler outputs
 preset = "cellprofiler_sqlite_pycytominer"
-# remove Image_Metadata_Plate from SELECT as this metadata was not extracted from file names
-# add Image_Metadata_Site as this is an important metadata when finding where single cells are located
-presets_config = """WITH Per_Image_Filtered AS (
+presets.config[preset][
+    "CONFIG_JOINS"
+    # remove Image_Metadata_Plate from SELECT as this metadata was not extracted from file names
+    # add Image_Metadata_Site as this is an important metadata when finding where single cells are located
```

Comment:
```suggestion
    # add Image_Metadata_FOV as this is an important metadata when finding where single cells are located
```
I don't see Site in this section so looks like FOV is more accurate.